### PR TITLE
Restore ability to create RSAKeys from components (from Vapor 2)

### PR DESCRIPTION
**Pull request by jseibert**

### CODE REVIEW

File: Sources/Crypto/RSA/RSAKey.swift [hunk 1]
```diff
@@ -39,6 +39,29 @@ public struct RSAKey {
         self.type = type
         self.c = key
     }
+    
+    /// Creates a new `RSAKey` from components.
```
Comment (tanner0101) [inline Sources/Crypto/RSA/RSAKey.swift]: A small example on how to use this API in the doc block would be great. 

See [this](https://github.com/vapor/crypto/blob/master/Sources/Crypto/Digest/Digest.swift#L106-L118) doc block for an example.

Comment (jseibert) [reply Sources/Crypto/RSA/RSAKey.swift]: @tanner0101 done!

**Comment by tanner0101** [comment]
Great, thanks for adding those comments @jseibert. One more thing: do you have access to test/sample value that we could parse in unit tests? That way we can make there aren't any regressions on this API going forward.

**Comment by jseibert** [comment]
@tanner0101 yes, I would just pull one of the google ones and hardcode these values in. They shouldn't expire:

```
{
    "kty": "RSA",
    "alg": "RS256",
    "use": "sig",
    "kid": "3b547886ff85a3428df4f61db73c1c23982a928e",
    "n": "mjJLokVSf3F_7MAPPEZzT0fQO2AQwlpzDdYG1EHH9WTxm0Dk4KB8vIBCp6lWm0fV8-pv0N7zF9rJ0CHKgkxuC02VwHVtuegE7XikfRCZJaPAn-MHm-eowW2SpSmsudi0_Gs1cvjxms_lVvoHUBaDTjhHWqCRGX_oOiNCglJKPFaYtyTA4ZiUfQ3FE_uVeoC_gYTYxuUzVxLsKJynrFaOVGIvnp9uRdbS0WtUhs7BY-tgqzJEt42_PFo-DAgWFIpdUzfG0AxAHZQ7TxDM09MaWBVoUMrBMqpMT6TaRtWiKOYeGEfV-ZH2d8qWoJHaKbZjMiSL64sgTNw2T_pZAyTI3Q",
    "e": "AQAB"
}
```

**Comment by tanner0101** [comment]
Awesome, thanks!

**Comment by jseibert** [comment]
@tanner0101 sounds good!

**Pull request merged**